Database/Buffer/Backend/index.ts

```diff
@@ -16,7 +16,6 @@ export class Backend {
 	private setAlarm = async () => {
 		!this.isAlarm &&
 			this.state.waitUntil(this.state.storage.setAlarm(Date.now() + (this.configuration?.snooze ?? 10000)))
-		return (this.isAlarm = true)
```

Comment:
Don't remove `this.isAlarm = true` this would break the alarm functionality.

Reply:
Also is it necessary to make this change together with the bump? Why is this part of the update to platform?